docs/source/openstack/volume-migration-using-s3.md

```diff
@@ -0,0 +1,190 @@
+# Volume migration use S3 API
+
+## Overview
+
+This article contains instructions for migration an OpenStack Cinder volume between two OpenStack environments (specifically, between the MOC Kaizen cluster and the NERC OpenStack cluster, but the instructions will work for any two OpenStack environments that have the necessary services enabled).
+
+Broadly, the process looks like this:
+
+1. Configure a `clouds.yaml` file with credentials for both environments
+1. Create EC2 credentials for both environments
+1. Create a backup of your volume on Kaizen
+1. Use the MinIO client to copy the backup data to the NERC
+1. Copy metadata for the backup from Kaizen to the NERC
+1. Create a volume to receive the backup on the NERC
+1. Restore the backup onto the new volume
+
+## Prerequisites
+
+### Create a clouds.yaml file
+
+You will need a [`clouds.yaml`](clouds.yaml) file that contains application credentials for both Kaizen and the NERC.
+
+- To create an application credential for Kaizen, see [this document](application-credentials).
+- To create an application credential for the NERC, see [the NERC openstack CLI documentation](https://nerc-project.github.io/nerc-docs/openstack/advanced-openstack-topics/openstack-cli/openstack-CLI/).
+
+In the remainder of this article, I'll assume you've named the two clouds `kaizen` and `nerc` in your `clouds.yaml` file.
+
+### Install the OpenStack CLI
+
+These instructions require use of the OpenStack command line client. Following the [installation guide][] to get the `openstack` command installed on your local system.
+
+[installation guide]: https://docs.openstack.org/newton/user-guide/common/cli-install-openstack-command-line-clients.html
+
+### Install MinIO client
+
+The [MinIO client][]is a CLI tool for interacting with S3-compatible object storage services.  The MinIO documentation has instructions for installing the client on your local system.
+
```

Comment:
Might be worth mentioning that you need to provide the full path `~/Downloads/mc.exe --help` instead of just `mc.exe --help` as the minIO documentation states. Not sure how tech-savvy our readers will be.